### PR TITLE
Feature/cal 015

### PR DESCRIPTION
**Pull request by alejandogodoy**

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ const express = require('express');
```
Comment (Alfondalix) [inline server/routes/index.js:4]: add const express = require('express');

File: server/routes/index.js [hunk 2]
```diff
@@ -2,6 +2,6 @@ const express = require('express');
 
 const router = express.Router();
```
Comment (Alfondalix) [inline server/routes/index.js:6]: try with, for example const boilerRouter = require('./boilers');
and place it above the code in line 3

File: server/routes/index.js [hunk 3]
```diff
@@ -2,6 +2,6 @@ const express = require('express');
...
 
-router.use('/', () => console.log('routes created'));
+router.use('/boilers', () => console.log('routes created'));
```
Comment (Alfondalix) [inline server/routes/index.js]: you should comment this line and add router.use('/boilers', boilerRouter);

File: server/models/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose");
```
Comment (Alfondalix) [inline server/models/index.js]: you should delete the index file

File: server/models/boilers.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+module.expots = mongoose => {
+    const Boilers = mongoose.model(
+        "boilers",
+        mongoose.Schema({
+            id: {
```
Comment (Alfondalix) [inline server/models/boilers.js]: the id is provided by mongoose, you should delete it

File: server/models/boilers.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+                type: mongoose.Schema.Types.ObjectId,
+                required: true
+                
+            },
+            description: {
+                type: String,
+                required: true
+            },
+            idType:{
+                type: mongoose.Schema.ObjectId,
```
Comment (Alfondalix) [inline server/models/boilers.js]: you should put the ref after thi line

File: server/controllers/boilers.js [hunk 7]
```diff
@@ -0,0 +1,106 @@
+const db = require("../models");
+const Boiler = db.boiler;
```
Comment (Alfondalix) [inline server/controllers/boilers.js]: try to change line 1 and 2 to const Boiler = require('..models/boilers.js');

**Review by Alfondalix — COMMENTED** [review]
those are the things or lines I would change, great work

**Review by Alfondalix — APPROVED** [review]

**Review by tomascravero — APPROVED** [review]

**Pull request merged**